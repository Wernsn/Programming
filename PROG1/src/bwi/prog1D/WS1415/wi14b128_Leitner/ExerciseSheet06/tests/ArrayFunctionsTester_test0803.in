8
5
abcdefgh
2000
1
1
11
bcdefgh
2010
5
5
12
cdefgh
2050
5
31
13
defgh
2090
12
31
14
efgh
1900
2
28
15
3
0
