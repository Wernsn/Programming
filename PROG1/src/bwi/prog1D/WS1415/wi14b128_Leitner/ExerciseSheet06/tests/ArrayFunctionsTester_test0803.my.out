sort 2: 
enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-28): enter number of attendees: [
abcdefgh, Saturday, January 1st 2000, 11
bcdefgh, Wednesday, May 5th 2010, 12
cdefgh, Tuesday, May 31st 2050, 13
defgh, Sunday, December 31st 2090, 14
efgh, Wednesday, February 28th 1900, 15
]
[
abcdefgh, Saturday, January 1st 2000, 11
bcdefgh, Wednesday, May 5th 2010, 12
cdefgh, Tuesday, May 31st 2050, 13
defgh, Sunday, December 31st 2090, 14
efgh, Wednesday, February 28th 1900, 15
]
